monai/apps/nuclick/transforms.py

```diff
@@ -0,0 +1,467 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import math
+import random
+
+import cv2
```

Comment:
Hi @finalelement ,

I think you should use optional import for `cv2`, refer to the pandas:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/data/iterable_dataset.py#L23

Thanks.

Reply:
The imports for both cv2 and skimage are optional now.